cms/views.py

```diff
@@ -48,3 +50,22 @@ class SearchMetaDetailView(SearchMetaDetailMixin, generic.DetailView):
 class PageDetailView(PageDetailMixin, generic.DetailView):
 
     """A simple page detail view."""
+
+
+class CacheMixin(object):
+    cache_timeout = 60
+
+    def get_cache_timeout(self):
+        return self.cache_timeout
+
+    def dispatch(self, *args, **kwargs):
+        dispatch = super(CacheMixin, self).dispatch
+
+        if getattr(settings, 'CMS_CACHE_PAGES', False):
```

Comment:
Should this be `if not getattr`? At the moment if you enable the caching system it'll disable it.